tests/cross/src/test/scala/tests/pc/CompletionSuite.scala

```diff
@@ -1289,13 +1235,80 @@ class CompletionSuite extends BaseCompletionSuite {
        |""".stripMargin
   )
 
-  checkItems(
-    "scope-completions-empty-query",
-    s"""|class Foo {
-        |  @@
+  check(
+    "ordering-1",
+    s"""|object Main {
+        |  languageFeature.@@
         |}
         |""".stripMargin,
-    _.nonEmpty
+    """|dynamics scala.languageFeature
+       |existentials scala.languageFeature
+       |experimental scala.languageFeature
+       |higherKinds scala.languageFeature
+       |implicitConversions scala.languageFeature
+       |""".stripMargin,
+    topLines = Some(5)
+  )
+
+  check(
+    "ordering-2",
+    s"""|object Main {
+        |  1.@@
+        |}
+        |""".stripMargin,
+    """|!=(x: Byte): Boolean
+       |!=(x: Char): Boolean
+       |!=(x: Double): Boolean
+       |!=(x: Float): Boolean
+       |!=(x: Int): Boolean
+       |!=(x: Long): Boolean
+       |!=(x: Short): Boolean
+       |%(x: Byte): Int
+       |%(x: Char): Int
+       |%(x: Double): Double
+       |""".stripMargin,
+    topLines = Some(10)
+  )
+
+  check(
+    "ordering-3",
+    s"""|class A {
+        |  def fooA: String = ""
+        |}
+        |
+        |class B extends A {
+        |  def fooB: String = ""
+        |}
+        |
+        |object Main {
+        |    val x = new B()
+        |    x.foo@@
+        |}
+        |""".stripMargin,
+    """|fooB: String
+       |fooA: String
+       |""".stripMargin,
+    topLines = Some(2)
+  )
+
+  // issues with scala 3 https://github.com/lampepfl/dotty/pull/13515
+  check(
+    "ordering-4".tag(IgnoreScala3),
```

Comment:
Could we ignore the current Scala 3 versions up to 3.1.0? So that we automatically check it for 3.1.1 ?